src/components/thumbnail/collapsible-document-section.tsx

```diff
@@ -66,7 +66,25 @@ export const CollapsibleDocumentsSection: React.FC<IProps> = observer(
       // get the learning logs
       networkResource.teachers?.forEach((teacher) => {
         if (teacher.learningLogs) {
-          for (const [key, document] of Object.entries(teacher.learningLogs)) {
+          for (const [key] of Object.entries(teacher.learningLogs)) {
+            documentKeys.push(key);
+          }
+        }
+      });
+    } else if (section.type === "published-personal-documents") {
+      // get the published personal documents
+      networkResource.resources?.forEach((resource) => {
+        if (resource.personalPublications) {
```

Comment:
The personalPublications are actually at the class level, not the offering level. That was one of my earlier mistakes. Also, you can handle the published learning logs from the class level if you're so inclined as well, although it's also fine to leave that for later.